site/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<!-- Mars convention -->
-		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_TIMESTAMP}-B${BUILD_NUMBER}/all/repo/</publishingPath>
 		<!-- Luna convention: TODO JBIDE-18772: remove this when master build only works for Mars -->
 		<publishingPath>builds/staging/${JOB_NAME}/all/repo/</publishingPath>
```

Comment:
Since Luna convention is no longer being used, and since we have the option to override in Jenkins, I've removed the line and updated it to use neon/ instead of mars/.

If you're still building in your 1.0.x branch/job, I'll need to apply this fix there too, but using mars/ instead of neon/ as the folder prefix.